typescript/sdk/src/metadata/agentConfig.ts

```diff
@@ -201,6 +255,13 @@ export const AgentChainMetadataSchema = ChainMetadataSchemaObject.merge(
       }
     }
 
+    // If the protocol type is Sealevel, require everything in AgentSealevelChainMetadataSchema
+    if (metadata.protocol === ProtocolType.Sealevel) {
+      if (!AgentSealevelChainMetadataSchema.safeParse(metadata).success) {
+        return false;
+      }
+    }
+
```

Comment:
i can't see where these new params are in the agent config?

Reply:
it's this merge https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/3a62b2df86efe6b6d3186266e6f689eb3020317d/typescript/sdk/src/metadata/agentConfig.ts#L212